Extension/src/LanguageServer/client.ts

```diff
@@ -312,23 +312,13 @@ class DefaultClient implements Client {
 
             // requests/notifications are deferred until this.languageClient is set.
             this.queueBlockingTask(() => languageClient.onReady().then(
-                () => {
+                async () => {
```

Comment:
We shouldn't mix `async` with `then`. Can the whole function be switched to `async`?

Reply:
Umm...this isn't really mixing async and then, because the lambda is async, right?

Reply:
You're using async inside the then.

Reply:
Yeah, that's just declaring the lambda as async -- the lambda itself just uses await and not any "then". Are you saying that it still counts as "mixing"?

Reply:
@bobbrow What do you want me to change?

Reply:
The only thing this change is trying to do is change the order of operations, right?  If that's the case you don't need to convert the lambda to async/await.

Reply:
I just changed the await to return the promise. We could switch everything to await later.